src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncUnorderedSpec .cs

```diff
@@ -29,24 +29,24 @@
 
 namespace Akka.Streams.Tests.Dsl
 {
-    public class FlowMapAsyncUnorderedSpec : AkkaSpec
+    public class FlowSelectAsyncUnorderedSpec : AkkaSpec
     {
         private ActorMaterializer Materializer { get; }
 
-        public FlowMapAsyncUnorderedSpec(ITestOutputHelper helper) : base(helper)
+        public FlowSelectAsyncUnorderedSpec(ITestOutputHelper helper) : base(helper)
         {
             Materializer = ActorMaterializer.Create(Sys);
         }
 
         [Fact]
-        public void A_Flow_with_MapAsyncUnordered_must_produce_future_elements_in_the_order_they_are_ready()
+        public void A_Flow_with_SelectAsyncUnordered_must_produce_future_elements_in_the_order_they_are_ready()
```

Comment:
future is used instead of task in all methods here, could make sense to rename it as well in the context of SelectAsyncUnordered